public/index.html

```diff
@@ -1,862 +1,1042 @@
-                                       rsathe emon</a></h3>
-                                 <span>Aug 28, 2017</span>
-                              </div>
-                           </div>
-                        </div>
-                     </div>
+                        <span>Aug 28, 2017</span>
+                      </div>
+                    </div>
                   </div>
-               </div>
-            </div>
-            <div class="bd-footer__copyright bd-border-top pt-40 pb-40">
-               <span>All rights reserved © 2022 <a href="index.html">eThemeStudio</a> </span>
+                </div>
+              </div>
             </div>
-         </div>
+          </div>
+          <div class="bd-footer__copyright bd-border-top pt-40 pb-40">
+            <span
+              ><a href="https://www.guisetech.com/" target="_blank">
+                © Guisetech Limited</a
+              >
+              , <span class="date"></span>, Registered in Turkiye. Guisetech
+              Limited (Registered Number (VKN): 4111087362). All Right Reserved.
+            </span>
+          </div>
+        </div>
       </div>
-   </footer>
-   <!-- footer area end -->
-
-
-   <!-- JS here -->
-   <script src="assets/js/vendor/jquery.js"></script>
-   <script src="assets/js/vendor/waypoints.js"></script>
-   <script src="assets/js/bootstrap-bundle.js"></script>
-   <script src="assets/js/meanmenu.js"></script>
-   <script src="assets/js/swiper-bundle.js"></script>
-   <script src="assets/js/magnific-popup.js"></script>
-   <script src="assets/js/backtotop.js"></script>
-   <script src="assets/js/nice-select.js"></script>
-   <script src="assets/js/counterup.js"></script>
-   <script src="assets/js/wow.js"></script>
-   <script src="assets/js/isotope-pkgd.js"></script>
-   <script src="assets/js/imagesloaded-pkgd.js"></script>
-   <script src="assets/js/ajax-form.js"></script>
-   <script src="assets/js/main.js"></script>
-</body>
+    </footer>
+    <!-- footer area end -->
 
+    <!-- JS here -->
+    <script src="assets/js/vendor/jquery.js"></script>
+    <script src="assets/js/vendor/waypoints.js"></script>
+    <script src="assets/js/bootstrap-bundle.js"></script>
+    <script src="assets/js/meanmenu.js"></script>
+    <script src="assets/js/swiper-bundle.js"></script>
+    <script src="assets/js/magnific-popup.js"></script>
+    <script src="assets/js/backtotop.js"></script>
+    <script src="assets/js/nice-select.js"></script>
+    <script src="assets/js/counterup.js"></script>
+    <script src="assets/js/wow.js"></script>
+    <script src="assets/js/isotope-pkgd.js"></script>
+    <script src="assets/js/imagesloaded-pkgd.js"></script>
+    <script src="assets/js/ajax-form.js"></script>
+    <script src="assets/js/main.js"></script>
+    <script>
+      let date = new Date().getFullYear();
+      let dom = (document.querySelector(".date").innerHTML = date);
```

Comment:
why did you define dom ? it is not being used anywhere. Check the scenario that; the query selector cannot find the item. What happens then?